### PR TITLE
Added instructions for running composer using docker

### PR DESCRIPTION
**Pull request by aepod**
## Purpose of this pull request

This PR adds instructions for running composer commands in a docker container, without having to use docker-compose. This is useful for CI/CD and instancing the application for the first time.

## Affected DevDocs pages

-  cloud/docker/docker-config.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/docker/docker-config.md [hunk 1]
```diff
@@ -89,6 +89,18 @@ Use the following command to stop and remove the Docker configuration:
 {: .bs-callout-warning}
 This command removes all components of your local Docker instance including containers, networks, volumes, and images except for the persistent database and the `magento-sync` volume. See [Rebuild a clean environment][refresh].
 
+## Running Composer with Docker
+
+It is possible to run composer using `docker` before you have instanced the containers. This is very useful for instancing the application during the CI/CD build process, or even first time Magento set up.
```
Comment (meker12) [inline src/cloud/docker/docker-config.md]:
```suggestion
You can run composer using `docker` commands before you create the containers. This technique is useful for creating an application instance during the CI/CD build process, or even first time Magento set up.
```

File: src/cloud/docker/docker-config.md [hunk 2]
```diff
@@ -89,6 +89,18 @@ Use the following command to stop and remove the Docker configuration:
...
+
+To do this you must use the proper [Docker Hub PHP Image Tag] that matches with the version of the Magento application. The following example shows PHP 7.3 and should be run in the project root directory.
```
Comment (meker12) [inline src/cloud/docker/docker-config.md]:
```suggestion
When you run composer with Docker commands, you must use the [Docker Hub PHP Image Tag] that matches the Magento application version. The following example uses PHP 7.3. Run the command from the project root directory. 
```

File: src/cloud/docker/docker-config.md [hunk 3]
```diff
@@ -89,6 +89,18 @@ Use the following command to stop and remove the Docker configuration:
...
+
+```bash
+docker run -it  -v $(pwd):/app/ -v ~/.composer/:/root/.composer/ magento/magento-cloud-docker-php:7.3-cli-1.1 bash -c "composer install&&chown www. /app/"
+```
+
+This command passes in the current working directory as `/app/`, includes composer from `~/.composer/` and runs the `composer install` in the container, after which it fixes the permissions on the files that have been added/changed.
```
Comment (meker12) [inline src/cloud/docker/docker-config.md]:
```suggestion
This command passes in the current working directory as `/app/`, includes composer from `~/.composer/`, and runs the `composer install` command in the container. After this set up, the command  fixes the permissions on the files that have been added or changed.
```

**Pull request merged**